extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -503,6 +514,16 @@ export class AmpStoryEmbeddedComponent {
       this.onOutsideTooltipClick_(event)
     );
 
+    this.tooltip_.addEventListener(
+      'click',
+      () =>
+        this.analyticsService_.triggerEvent(
```

Comment:
Hmm not for now since it's inside a shadow root. It won't be recognized by the `selector`. If we were to support `selector` inside a shadow root then we would have to do that. I'll just add it, just in case :) PTAL